eswifi: build failure

**Describe the bug**
```
<drivers/wifi/eswifi/eswifi_socket_offload.c>: In function 'eswifi_socket_poll':
<drivers/wifi/eswifi/eswifi_socket_offload.c>:448:24: error: 'POLLIN' undeclared (first use in this function)
  448 |  if (!(fds[0].events & POLLIN)) {
      |                        ^~~~~~
<drivers/wifi/eswifi/eswifi_socket_offload.c>:448:24: note: each undeclared identifier is reported only once for each function it appears in
<drivers/wifi/eswifi/eswifi_socket_offload.c>: In function 'eswifi_off_getaddrinfo':
<drivers/wifi/eswifi/eswifi_socket_offload.c>:621:9: error: 'EAI_MEMORY' undeclared (first use in this function); did you mean 'DNS_EAI_MEMORY'?
  621 |   err = EAI_MEMORY;
      |         ^~~~~~~~~~
      |         DNS_EAI_MEMORY
make[2]: *** [zephyr/CMakeFiles/zephyr.dir/drivers/wifi/eswifi/eswifi_socket_offload.c.obj] Error 1
make[1]: *** [zephyr/CMakeFiles/zephyr.dir/all] Error 2
make: *** [all] Error 2
```

**To Reproduce**
Steps to reproduce the behavior:
1. sanitycheck -b -p disco_l475_iot1 -T samples/net/wifi/

--- a/drivers/wifi/eswifi/eswifi_socket_offload.c
+++ b/drivers/wifi/eswifi/eswifi_socket_offload.c
@@ -445,7 +445,7 @@ static int eswifi_socket_poll(struct zsock_pollfd *fds, int nfds, int msecs)
 		return -1;
 	}
 
-	if (!(fds[0].events & POLLIN)) {
+	if (!(fds[0].events & ZSOCK_POLLIN)) {
 		errno = ENOTSUP;
 		return -1;
 	}
@@ -464,7 +464,7 @@ static int eswifi_socket_poll(struct zsock_pollfd *fds, int nfds, int msecs)
 		return -1;
 	}
 
-	fds[0].revents = POLLIN;
+	fds[0].revents = ZSOCK_POLLIN;
 
 	/* Report one event */
 	return 1;
@@ -618,7 +618,7 @@ static int eswifi_off_getaddrinfo(const char *node, const char *service,
 	*res = calloc(1, sizeof(struct zsock_addrinfo));
 	ai = *res;
 	if (!ai) {
-		err = EAI_MEMORY;
+		err = DNS_EAI_MEMORY;
 		goto done_unlock;
 	}
 
@@ -626,7 +626,7 @@ static int eswifi_off_getaddrinfo(const char *node, const char *service,
 	ai_addr = calloc(1, sizeof(*ai_addr));
 	if (!ai_addr) {
 		free(*res);
-		err = EAI_MEMORY;
+		err = DNS_EAI_MEMORY;
 		goto done_unlock;
 	}
 

